docs/algorithms/sig/dilithium.yml

```diff
@@ -15,22 +15,16 @@ website: https://pq-crystals.org/dilithium/
 nist-round: 3
 spec-version: 3.1
```

Comment:
Looks fishy to me: No change to spec version but different KATs?

Reply:
updated in 743c15ceea63ac88631d1ec16bab1ea6061eb6f0